blog-posts/thoughs-on-collaborative-work.md

```diff
@@ -0,0 +1,81 @@
+---
+path: "/blog/thoughts-on-collaborative-work/"
+date: "2022-02-24"
+title: "Thoughts on collaborative work"
+author: Jesús Real
+authorSummary: Senior Frontend Engineer @ Satellytes
+featuredImage: images/thoughs-on-collaborative-work/group-table.jpg
+attribution:
+    creator: Marvin Meyer
+    source: https://unsplash.com/photos/SYTO3xs06fU
+seoMetaText: This article aims to offer an overview about the common beliefs around collaborative work, highlight the (many) (surprising) positive sides of it, and depicts some on-field examples that will either suit you or inspire you to find your own ways on approaching collaborative work. 
+teaserText: This article aims to offer an overview about the common beliefs around collaborative work, highlight the (many) (surprising) positive sides of it, and depicts some on-field examples that will either suit you or inspire you to find your own ways on approaching collaborative work.
+leadboxText: Are you excited about getting the most out your team work? Would you like to work with great professionals? Then join us!
+
+
+---
+
+Most of us developers, despite being more or less aware of the benefits of doing more work together, work on our own most of the time. Why? What can we do about it? Let's dive into it!
+
+## Is it just about programming?
+
+Short answer, no, it’s not. As software developers we don’t just write code, we also play a crucial role in actively shaping, challenging and clarifying requirements. And this happens in many different flavours: emails, issue tracking systems, chat tools, team meetings, inter-team meetings... Every area of our work can benefit if we approach it in a more collaborative way.
+
+## The good, the bad and the ugly. Or is it just good?
+
+What is the greatest hesitance when thinking about working more with other colleagues? Well, most of us think it is generally a waste of time, that it is an inefficient way to work, that there is no time for that. But if we dig deeper, we will find two strong causes. First, what I call "the developer pride". We think we are great if we solve problems on our own, if we do it without help, but if we ask for help it means we are not good enough. Second, shame. Often times we feel like what we do is not good enough to be shared. The idea of someone watching you work, the typos you do, the uncertainty with which you approach problems, etc. is frightening for most of us. Literally “stage fright”.
```

Comment:
>  We think we are great if we solve problems on our own,

You could describe that as "we all have a good feeling when doing things on our own" instead of implying actual thoughts about being better or worse. I think it's often not such a direct feeling.

Reply:
I agree. I softened it and removed the wording "Pride" and "Shame".